src/prefect/runtime/task_run.py

```diff
@@ -112,5 +120,6 @@ def get_parameters() -> Dict[str, Any]:
     "tags": get_tags,
     "name": get_name,
     "parameters": get_parameters,
+    "run_count": get_run_count,
```

Comment:
nitpick: it seems like we were documenting these values in the docstring at the top of the file, do you think we should add these there?

Reply:
IMO, `FIELDS` is necessary to find the function to call when a named attribute is accessed. If anything,`__all__` maybe isn't totally necessary but it makes it clear what can be imported directly.

Reply:
[This is the docstring](https://github.com/PrefectHQ/prefect/blob/daebb1b9473cba1f720c12670d423b8da4c71a87/src/prefect/runtime/task_run.py#L9) that I'd noticed. I don't think it's up to date right now but like you mentioned, the `FIELDS`/`__all__` makes it clear what can be used.

Reply:
Ah good catch, let me fix that since the docstring is important for our [API ref](https://docs.prefect.io/latest/api-ref/prefect/runtime/flow_run/).

Reply:
Updated in [29ee2fb](https://github.com/PrefectHQ/prefect/pull/10676/commits/29ee2fb3c070e2a5457708acd89e1b5377d5eb44)

Reply:
I totally misunderstood your initial comment, I realize now 😅